dockerfiles/theia/e2e/src/docker-run.sh

```diff
@@ -9,10 +9,16 @@
 echo "Starting Theia..."
 rm -rf /root/logs/*
 HOME=/home/theia /entrypoint.sh > /root/logs/theia.log 2>/root/logs/theia-error.log&
-sleep 10s
+
 echo "Cleaning videos folder..."
 # Cleanup previous videos
-rm -rf /root/cypress/videos
+rm -rf /root/cypress/videos/*
+
+# Find TCP 0.0.0.0:3100 that will be opened by Theia.
+until `ss -nlt | grep -Fq ':3100'`; do
+    echo "Waiting for booting up Theia..."
+    sleep 10s
```

Comment:
I think the wait increment is too long no ?

Reply:
It requires 40 minutes to run each CI build (on my server. Ubuntu 16.10 / Azure D2 VM).
IMO, it won't effective to check more frequently.

Reply:
If you already have the image pre-built, and that you launch the script it should be fast. Here it's waiting 10s on each increment which is too long as you enhance the script to not wait so long.
IMHO should be around 1-2s

Also I think we should add a MAX_WAIT parameter to avoid infinite loop (let say theia is failing then it won't start then it won't listen and e2e will wait for eternity)

Reply:
I'm easy about the value of interval.
Since it's hard to boot Theia up within 1 or 2 second, I'll fix to 5 sec as the initial and 2 second in the loop.

I agree the limit of the loop is required.